### PR TITLE
Implement geocoding/reverse-geocoding/date & time parse/guestlist functionality

### PR DESCRIPTION
**Pull request by hecwon**

### CODE REVIEW

File: frontend/components/events/Guests.jsx [hunk 1]
```diff
@@ -2,26 +2,37 @@ import React, { useState } from 'react';
...
+const Guests = ({invitees, setInvitees}) => {
+  const navigation = useNavigation();
   const sampleData = userData;
-  const [invitees, setInvitees] = useState([]);
 
   const handleInvite = (guestId) => {
     const userIdInList = invitees.indexOf(guestId);
     if (userIdInList !== -1) {
       invitees.splice(userIdInList, 1);
+      console.log(invitees);
```
Comment (kallycao) [inline frontend/components/events/Guests.jsx:15]: Please omit this console.log!

File: frontend/components/events/Guests.jsx [hunk 2]
```diff
@@ -2,26 +2,37 @@ import React, { useState } from 'react';
...
+
   return (
     <ScrollView>
+      <Button title='Confirm Guests' onPress={handleConfirm}/>
       { sampleData.map((each) => {
         return (
-          <ListItem key={each.id} style={{ backgroundColor: ((!!invitees.indexOf(each.id)) ? '#2D70F9' : 'white' )}}>
+          <ListItem
+            key={each.id}
+            style={ invitees.indexOf(each.id) === -1 ? styles.unselectedGuest : styles.selectedGuest }
```
Comment (kallycao) [inline frontend/components/events/Guests.jsx:34]: Love your incorporation of styling with the conditional ternary operator

File: frontend/components/events/PendingEvents.jsx [hunk 3]
```diff
@@ -27,38 +29,57 @@ const PendingEvents = ({DYNAMICUSERINFO}) => {
...
+        };
+        useEffect(() => {
+          Location.reverseGeocodeAsync(geocodingInput)
+          .then((geocodedAddress) => {
+            setAddress(
+              `${geocodedAddress[0].streetNumber ? `${geocodedAddress[0].streetNumber} ` : ''}${geocodedAddress[0].street}, ${geocodedAddress[0].city} ${geocodedAddress[0].region} ${geocodedAddress[0].postalCode}`
+            );
+          })
+          .catch((err) => {
+            console.log(err);
```
Comment (KateFeaster) [inline frontend/components/events/PendingEvents.jsx:52]: Remove console.log. AJAX errors should only be caught on the front end if they're going to cause the request to retry or some other flow of logic

File: frontend/components/events/createEvent.jsx [hunk 4]
```diff
@@ -1,15 +1,18 @@
...
-import { useNavigation } from '@react-navigation/native';
+import { useNavigation, route } from '@react-navigation/native';
+import * as Location from 'expo-location';
 
 
-const CreateEvent = ({setCreate, DYNAMICUSERINFO}) => {
+const CreateEvent = ({invitees, DYNAMICUSERINFO}) => {
   const navigation = useNavigation();
   const sampleData = userData;
+  console.log(invitees);
```
Comment (KateFeaster) [inline frontend/components/events/createEvent.jsx:15]: Please remove console.log

Comment (kallycao) [inline frontend/components/events/createEvent.jsx:15]: Please omit this console.log

File: frontend/components/events/createEvent.jsx [hunk 5]
```diff
@@ -18,14 +21,44 @@ const CreateEvent = ({setCreate, DYNAMICUSERINFO}) => {
...
     date: (new Date()),
     address: null,
-    invitees: []
+    invitees: null
   })
 
+  useEffect(() => {
+    event.invitees = invitees;
+    setEvent({...event});
+    console.log('event with updated invitees: ', event);
```
Comment (kallycao) [inline frontend/components/events/createEvent.jsx:30]: Please omit this console.log

**Review by KateFeaster — CHANGES_REQUESTED** [review]
Couple of console logs to clear up, then gtg!

File: frontend/components/events/createEvent.jsx [hunk 6]
```diff
@@ -18,14 +21,44 @@ const CreateEvent = ({setCreate, DYNAMICUSERINFO}) => {
...
+
+  const guestlist = invitees.map((each) => {
+    for (var i = 0; i < userData.length; i++) {
+      console.log('i', i, 'userData: ', userData[i]);
+      if (userData[i].id === each) {
+        return userData[i];
+      }
+    }
+  })
+  console.log('guest list: ', guestlist);
```
Comment (kallycao) [inline frontend/components/events/createEvent.jsx:50]: Please omit this console.log

**Review by kallycao — APPROVED** [review]
Amazing work Hector!  I understand console.logs are super handy during development phase, so I don't believe it's an issue to leave them in during this phase. Just make sure to omit them in your final production code.

File: frontend/components/events/createEvent.jsx [hunk 7]
```diff
@@ -41,24 +40,28 @@ const CreateEvent = ({invitees, DYNAMICUSERINFO}) => {
...
   })
-  console.log('guest list: ', guestlist);
 
-  const coordinatify = () => {
-    Location.geocodeAsync(event.address)
+  const coordinatify = async () => {
+    await Location.geocodeAsync(event.address)
     .then((results) => {
-      console.log(results);
+      event.address = [results[0].latitude, results[0].longitude];
```
Comment (KateFeaster) [inline frontend/components/events/createEvent.jsx:52]: Don't modify state variables. Make a copy, the modify the variable

**Review by KateFeaster — APPROVED** [review]

**Pull request merged**